src/blob.rs

```diff
@@ -1,12 +1,55 @@
-use std::path::Path;
+use futures::{Async, Poll};
+use std::{
+    io,
+    marker::PhantomData,
+    path::{Path, PathBuf},
+};
+use tokio::{
+    fs::file::{File, OpenFuture, OpenOptions},
+    prelude::Future,
+};
 
 use crate::record::Record;
 
 /// A `Blob` struct for performing of database,
 #[derive(Debug, Default)]
-pub struct Blob<T> {
+pub struct Blob<K>
+where
+    K: Send,
```

Comment:
Maybe `Default + Send`?
And probably it's time to create a trait for key that inherits from all required traits (currently `Default` and `Send`)

Reply:
It is outdated now